configuration/transitions.md

```diff
@@ -39,6 +40,30 @@ In this example the `d` transition will not be applied, but the other three will
 
 Guides for how to setup specific transitions.
 
+### multi_form_alerts
+
+Full documentation in [kanso.json](https://github.com/medic/medic-webapp/blob/master/kanso.json) or in the Dashboard app_settings page.
```

Comment:
Dashboard page to edit app_settings is not going away yet, we're keeping it for now. It contains the fullest documentation on app_settings. So I'm adding to that because it's where the doc lives for now, moving it out is another problem.